File: skidbacken/submissions/accepted/joshua.py
#!/usr/bin/python3

r,c = [int(i) for i in input().split()]

grid = []
for i in range(r):
    grid.append(input())

lanes = 0
for i in range(c):
    works = True
    for j in range(r):
        if grid[j][i] == '#':
            works = False
    
    if works:
        lanes += 1
            
print(lanes)

File: jakobwidebrant/submissions/accepted/joshua.cpp
#include <bits/stdc++.h>

using namespace std;


int main()
{
	int r, c;
	cin >> r >> c;

	vector<string> grid(r);
	
	for (int i = 0; i < r; i++) cin >> grid[i];

    int lanes = 0;
    for (int i = 0; i < c; i++)
    {
        bool works = true;
        for (int j = 0; j < r; j++) if (grid[j][i]=='#') works = false;
        if (works) lanes++;
    }

	cout << lanes << "\n";

	return 0;
}